src/scss/objects/_form-toggle.scss

```diff
@@ -0,0 +1,166 @@
+$formToggle-button-color            : $blue;
+$formToggle-button-background       : $blue--offWhite;
+
+.o-formToggle {
+    position: relative;
+    display: inline-block;
+    margin-bottom: spacing();
+    border-radius: $formToggle-border-radius;
+    padding: ($formToggle-padding / 2) $formToggle-padding;
+    border: $formToggle-border-width solid $formToggle-border-color;
+
+    @include media('>=mid') {
+        background: $formToggle-background;
+    }
+}
+
+    .o-formToggle--button {
+        padding-left: spacing(x4);
+        color: $formToggle-button-color;
+        background: $formToggle-button-background;
+        border-color: $formToggle-button-background;
+    }
+
+    .o-formToggle--buttonIcon {
+        top: 50%;
+        position: absolute;
+        left: $formToggle-padding;
+        transform: translateY(-50%);
+        fill: $formToggle-button-color;
+    }
+
+    .o-formToggle--fullWidth {
+        width: 100%;
+        display: block;
+
+        @include media('>=mid') {
+            display: inline-block;
+        }
+    }
+
+    .o-formToggle--withCount {
+        padding-right: 15px;
+    }
+
+    .o-formToggle--withBg {
+        background: $formToggle-background;
+    }
+
+    .o-formToggle--mediumSized {
+        padding: $formToggle-padding ($formToggle-padding * 1.5);
+    }
+
+    .o-formToggle--largeSized {
+        height: $formToggle-large-size;
+        width: calc(50% - 4px);
+
+        &:nth-child(2n) {
+            margin-left: spacing();
+        }
+    }
+
+    .o-formToggle--largeAboveMid {
+
+        @include media('>=mid') {
+            float: left;
+            width: calc(50% - 4px);
+            height: $formToggle-large-size;
+
+            &:nth-child(2n) {
+                margin-left: spacing();
+            }
+        }
+    }
+
+    .o-formToggle-text {
+        display: block;
+        transition: padding 200ms ease-in-out;
+
+        &:after,
+        &:before {
+            content: '';
+        }
+
+        &:before {
+            top: 50%;
+            opacity: 0;
+            width: 12px;
+            height: 5px;
+            position: absolute;
+            margin-top: -4px;
+            left: spacing();
+            display: inline-block;
+            transform: rotate(-45deg) scale(0.5);
+            border-left: 1px solid $formToggle-tick-background;
+            border-bottom: 1px solid $formToggle-tick-background;
+            transition: transform 200ms ease, opacity 250ms ease;
+        }
+
+        .o-formToggle-input:checked ~ & {
+
+            padding-left: 20px;
+            font-weight: $font-weight-bold;
+
+            &:before {
+                opacity: 1;
+                transform: rotate(-45deg) scale(1);
+            }
+        }
+
+        .o-formToggle:hover &,
+        .o-formToggle:focus &,
+        .o-formToggle-input:hover ~ &,
+        .o-formToggle-input:focus ~ &,
+        .o-formToggle-input:checked ~ & {
+
+            &:after {
```

Comment:
What does this do? Can you add some comments to this? (not sure what `calc(100% + 2px)` is trying to do – prob obvious, but it's late on a Friday and Jon is making me review this ;)

Reply:
This creates a border over the top of it's parent. It's so we can add a border to the parent without adding a class with JS - nice and light :) will add comments!